target/linux/realtek/files-6.6/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -459,7 +459,8 @@ static int rtl93xx_pcs_validate(struct phylink_pcs *pcs,
 		phylink_set(mask, 10000baseCR_Full);
 	}
 
-	if (state->interface == PHY_INTERFACE_MODE_USXGMII) {
+	if (state->interface == PHY_INTERFACE_MODE_USXGMII ||
+	    state->interface == PHY_INTERFACE_MODE_10GBASER) {
```

Comment:
How does 10GBase-R support 2500Base-T and 5000Base-T?! Isn't it rather the PHY performing rate adaptation then and it should be up to phylink validate the available link modes?

Reply:
Yes, this patch is utter bullshit. A result of blind experiments before I realized that my real problem was the bogus virtual "phy-handle", which would hide the real interface from the driver.  It has no effect after that issue is resolved

Will drop this patch and rebase.  Thanks for the review, and sorry for including nonsense like this!